src/test/resources/pages/adminEmailCompose.html

```diff
@@ -347,7 +344,7 @@
                                 </span>
                             </form>
     
-                            <div id="documentBaseUrl">${app.url}</div>
+                            {*}
```

Comment:
If the app url is empty, it fails because the wildcard doesn't compare with empty strings